include/exadg/incompressible_navier_stokes/spatial_discretization/spatial_operator_base.h

```diff
@@ -241,12 +242,18 @@ class SpatialOperatorBase : public dealii::Subscriptor
   dealii::AffineConstraints<Number> const &
   get_constraint_p() const;
 
+  dealii::AffineConstraints<Number> const &
+  get_constraint_u() const;
+
   dealii::types::global_dof_index
   get_number_of_dofs() const;
 
   double
   get_viscosity() const;
 
+  SpatialDiscretization const &
```

Comment:
Could you explain why this member function is needed?

Reply:
I need it here
https://github.com/exadg/exadg/blob/b82815944e73824ce05b1623c36fc89785958891/include/exadg/incompressible_navier_stokes/time_integration/time_int_bdf.cpp#L617-L618
I also started using it in `spatial_operator_base_cpp` after I introduced it, but I'll make sure it's consistent and not a mixture of `param.spatial_discretization` and `get_spatial_discretization()`.

Reply:
Yes, you could directly ask param (we generally rely on that in the time integration classes in order to avoid writing many getter functions in the spatial discretization classes).

Reply:
@NiklasWik also here (and for other comments) it would be great if you could point me directly to your most recent changes.

Reply:
This was removed here https://github.com/exadg/exadg/pull/233/commits/dbfbaf9f1489c35dff9d9de8c34338011780e4ff#diff-0cd2dc6d304bbc5350a7fb4cf865ac674a9b35b07d2b3463fc7a887f0943c2e6